### ISSUE TITLE
Can't install latest version of docusaurus with npm

### ISSUE THREAD

**Issue by qrevel** [facebook/Docusaurus#692]
### Is this a bug report?

yes

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

yes

### Environment

Node 10+
Docusaurus 1.1.2

### Steps to Reproduce

```
docker run -it --rm node:10.1-alpine sh
/ # npm install docusaurus@1.1.2
npm ERR! code ETARGET
npm ERR! notarget No matching version found for docusaurus@1.1.2
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-05-23T05_54_54_343Z-debug.log
/ # npm show docusaurus | grep 1.1.2
     gaze: '^1.1.2',
/ #
```

### Expected Behavior

It should install the lastest version of docusaurus.
I found this [incident](https://status.npmjs.org/incidents/3rjzvts6v98k) on npm. It may be related.

**Comment by yangshun** [comment]
We received a similar report and it seems like rxjs is having the same issue as well https://github.com/ReactiveX/rxjs/issues/3729 and issues have been filed with npm https://github.com/npm/npm/issues/20711. 

We'll monitor the situation and perhaps publish 1.1.3 if it persists.

**Comment by JoelMarcey** [comment]
npm claims to have resolved this: https://status.npmjs.org/incidents/3rjzvts6v98k

Is it still an issue for you @qrevel?

**Comment by yangshun** [comment]
Closed by #615

**Comment by qrevel** [comment]
Everything's working with 1.1.3
